ansible/async_provision.yml

```diff
@@ -5,7 +5,7 @@
   become: yes
 
   vars:
-    var_bootstrap_version: "{{ bootstrap_version | default('stable', true) }}"
+    var_bootstrap_version: "{{ bootstrap_version | default('v2.0.1, true) }}"
```

Comment:
let's make `master` the default, or remove the default at all.

The hardcoded version here will only have us headaches